lib/erlef/data/resources.ex

```diff
@@ -1,19 +1,24 @@
-defmodule Erlef.Community.Resources do
+defmodule Erlef.Data.Resources do
   @moduledoc """
-  Module for getting static community data
+  Module for getting static resources for Community or Affiliates
 
   All resources data for the community page of the site can be found in 
   [priv/data/community](priv/data/community). 
+
+  All resources data for the affiliate page of the site can be found in 
+  [priv/data/community](priv/data/affiliate). 
+
   Said data ends up compiled into this module along with a few dynamically generated helper functions.
+
   As an example,we have all active languages in 
   `priv/data/community/languages.exs`, 
   this file ends up being evalulated and the base name of the file without the 
   extension (i.e., `"languages"`) ends up being used to create a helper function called `all_languages/0`. 
   Like wise the atom `languages` is also used as a key pointing to the evalulated term as returned by the `all/0`  function. 
   """
 
-  data =
-    for d <- "priv/data/community/*.exs" |> Path.wildcard() |> Enum.sort() do
+  load_data = fn path ->
```

Comment:
Instead of putting the affiliates data in the `Erlef.Community.Resources` module we should probably just make a  `Erlef.Community.Affiliates` module. It's ok to copy the functionality for pulling in the data from the community module into the affiliates module. Rationale is the [rule of three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)). Thus, this module should be renamed back to `Erlef.Community.Resources` and have `Erlef.Community.Affiliates`  

I think after that's done and some tests are updated, then we should be good to go 🚀

Reply:
I believe this file needs to be deleted.